.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 .DS_Store
 .cache
+.vscode
```

Comment:
We don't add editor specific files to this, you can set up a global ignore locally.